docs/api.md

```diff
@@ -693,17 +693,18 @@ follow a few rules when writing asynchronous templates:
 
 ## Autoescaping
 
-By default, nunjucks will render all output as it is. If you turn on
-autoescaping, nunjucks will escape all output by default. It's
-recommended that you do this for security reasons.
+By default, nunjucks will escape all output. It's recommended
+that you do this for security reasons. If you turn off autoescaping,
+nunjucks will render all output as it is by default.
 
-Autoescaping is rather simplistic in nunjucks right now. All you have
-to do is pass the `autoescape` option as `true` to the `Environment`
-object. In the future, you will have more control over which files
-this kicks in.
+Autoescaping is rather simplistic in nunjucks right now. In the future,
```

Comment:
If we're updating this doc anyway, let's just remove the sentence about "in the future..." No need to make promises that nobody (AFAIK) currently has plans to implement.